### ISSUE TITLE
ember-template-lint "overrides" config does not work with eslint-plugin-hbs

### ISSUE THREAD

**Issue by tstewart15** [ember-template-lint/eslint-plugin-hbs#37]
I'm trying to override specific ember-template-lint rules for certain files/directories in my repo, using the ember-template-lint ["overrides" config prop](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/overrides.md#sample-configuration).

This does not seem to work.

Is there another method of overriding ember-template-lint rules for specific files/directories that works with eslint-plugin-hbs? Or do we need to fix this issue?

**Comment by rwjblue** [comment]
Ya, I think we'll need to make changes to address.

**Comment by jaydgruber** [comment]
I ran into this issue where I wanted to be able to enable [no-bare-strings](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-bare-strings.md) on app templates files, but ignore it on inline templates in test files.

I ended up doing the following:
```javascript
// .template-lintrc.js
module.exports = {
  rules: {
    'no-bare-strings': false,
  },
  overrides: [
    {
      files: ['**/*.hbs'],
      rules: {
        'no-bare-strings': true,
      }
    },
  ],
};
```

**Comment by tstewart15** [comment]
@rwjblue The issue does not appear fixed after upgrading to version 2.13.0. Setting the following does not correctly ignore the listed overrides files:
```js
module.exports = {
  rules: {
    'no-bare-strings': true,
  },
  overrides: [
    // Allow non-translated strings in fixtures/ and HealthCheck
    {
      files: [
        '**/test-js/glimmer/fixtures/**',
        '**/HealthCheck.js',
      ],
      rules: {
        'no-bare-strings': false,
      },
    },
  ],
};
```
